Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM php:7.2-apache-stretch
+
+WORKDIR /var/www/html
+
+COPY ./docker/000-default.conf /etc/apache2/sites-available/
+COPY ./docker/ports.conf /etc/apache2/
+
+EXPOSE 8080
+
+# work around https://github.com/docker-library/openjdk/blob/0584b2804ed12dca7c5e264b5fc55fc07a3ac148/8-jre/slim/Dockerfile#L51-L54
+RUN mkdir -p /usr/share/man/man1
+
+RUN apt-get update -q && apt-get install -y \
+        epubcheck \
+        libzip-dev \
+        unzip \
+        wget \
```

Comment:
Looks like this also needs `libgl1-mesa-glx` (in my testing, that seems to get rid of the errors; there are still 38 warnings).

Reply:
Actually, sorry, scratch that, it should be these two (as listed in the Calibre install docs):

```
xdg-utils \
xz-utils \
```

Reply:
I've added both. Thanks